src/components/Header/Header.test.jsx

```diff
@@ -4,6 +4,7 @@ import { Router } from 'react-router-dom';
 import { createMemoryHistory } from 'history';
```

Comment:
Here I would suggest testing the search functionality if possible because is the main feature of this component.